hexrd/ui/main_window.py

```diff
@@ -326,7 +326,7 @@ def open_grain_fitting_results(self):
             HexrdConfig().working_dir = str(path.parent)
 
             data = np.loadtxt(selected_file)
-            dialog = FitGrainsResultsDialog(data)
+            dialog = FitGrainsResultsDialog(data, full_workflow=False)
```

Comment:
Rename `full_workflow` to `allow_export_workflow`

Reply:
Changed